ping_one/manifest.json

```diff
@@ -2,20 +2,25 @@
   "manifest_version": "2.0.0",
   "app_uuid": "5fce4e84-ab6e-4ba6-b996-ac85e134642e",
   "app_id": "ping-one",
-  "display_on_public_website": false,
+  "display_on_public_website": true,
   "tile": {
     "overview": "README.md#Overview",
     "configuration": "README.md#Setup",
     "support": "README.md#Support",
     "changelog": "CHANGELOG.md",
-    "description": "Gain insights into PingOne logs",
-    "title": "ping_one",
-    "media": [],
+    "description": "Gain insights into PingOne logs.",
+    "title": "PingOne",
+    "media": [
+      {
+        "caption": "PingOne",
+        "image_url": "images/ping_one.png",
+        "media_type": "image"
+      }
+    ],
     "classifier_tags": [
       "Category::Log Collection",
       "Category::Security",
-      "Submitted Data Type::Logs",
-      "Offering::Integration"
```

Comment:
Was removing `Offering::Integration` intentional?

Reply:
Do we need to add the OS Tags?

Reply:
@nathanmadams Do we add OS Tags for crawler based Integrations?

Reply:
> Was removing `Offering::Integration` intentional?

Nope, I have reverted it back